rca/landingpages/models.py

```diff
@@ -549,12 +560,68 @@ def get_context(self, request, *args, **kwargs):
         context = super().get_context(request, *args, **kwargs)
         context["featured_image_secondary"] = self.get_featured_image_secondary()
         context["page_list"] = self.get_page_list()
-
+        if self.tap_widget:
+            context["tap_widget_code"] = mark_safe(self.tap_widget.script_code)
```

Comment:
**Issue**
I think this will throw an attribute error if no tap widget is set, as tap_widget here could be a NoneType if not set